android/app/build.gradle

```diff
@@ -34,18 +40,24 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.stadtnavi_app"
+        applicationId "com.test.stadtnaviApp"
         minSdkVersion 16
         targetSdkVersion 30
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
     }
 
+    signingConfigs {
+        release {
```

Comment:
Could we have here an if-else statement, so that we have still the possibility to build it without a Keystore file like here:

```
 if (keystorePropertiesFile.exists()) {
       keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
   }
```

https://flutter.dev/docs/deployment/android#configure-signing-in-gradle